src/lib/styles/mixins.sass

```diff
@@ -31,3 +31,10 @@
   border-radius: var(--radius)
   box-shadow: 1px 1px 5px 0 rgba(0, 0, 0, 0.02), 1px 1px 15px 0 rgba(0, 0, 0, 0.03)
   transition: transform var(--transition-normal), background-color var(--transition-normal), box-shadow var(--transition-slow)
+
+@mixin image-hover-animation
+  transition: transform var(--transition-fast) ease-in-out
+  &:hover
+    transform: scale(1.1)
+  &:focus
```

Comment:
I would drop focus, hover should suffice

Reply:
Done